script/babel-transform-constants.mjs

```diff
@@ -1,27 +1,62 @@
-module.exports = transform
+export default transform
 
-var path = require('path')
-var resolveFrom = require('resolve-from')
+import path from 'path'
+import resolveFrom from 'resolve-from'
+import codes from '../lib/character/codes.js'
+import values from '../lib/character/values.js'
+import constants from '../lib/constant/constants.js'
+import types from '../lib/constant/types.js'
 
 var supported = [
-  require.resolve('../lib/character/codes'),
-  require.resolve('../lib/character/values'),
-  require.resolve('../lib/constant/constants'),
-  require.resolve('../lib/constant/types')
+  'micromark/lib/character/codes.js',
+  'micromark/lib/character/values.js',
+  'micromark/lib/constant/constants.js',
+  'micromark/lib/constant/types.js'
 ]
 
-var evaluated = supported.map(function (filePath) {
-  return require(filePath)
-})
+var evaluated = [codes, values, constants, types]
 
 function transform() {
   return {
     visitor: {
+      ImportDeclaration: ImportDeclaration,
       VariableDeclaration: VariableDeclaration,
       MemberExpression: MemberExpression
     }
   }
 
+  function ImportDeclaration(p, state) {
+    var dirname = path.dirname(state.filename)
+    var id
+    var actual
+    var local
+    var position
+
+    actual = resolveFrom(dirname, p.node.source.value)
+    p.node.specifiers.forEach((specifier) => {
+      if (specifier.type === 'ImportDefaultSpecifier') {
+        id = specifier.local.name
+      } else {
+        throw Error(
+          'Unknown specifier "' + specifier.type + '" in "' + p.toString() + '"'
```

Comment:
```suggestion
          'Unknown specifier "' + specifier.type + '" in "' + String(p) + '"'
```

doubt it would happen but this prevents `undefined.toString()` throwing weird errors